riotfile.py

```diff
@@ -2604,7 +2604,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             pkgs={"requests": latest, "gevent": latest},
             venvs=[
                 Venv(
-                    pys=select_pys(min_version="3.5"),
+                    pys=select_pys(min_version="3.8", max_version="3.10"),
```

Comment:
Python 3.11 is not yet supported? Perhaps worth adding a brief comment explaining why we are doing this?